src/Kirel.Repositories/Persistance/PredicateBuilder.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Linq.Expressions;
```

Comment:
Install the Kirel.Shared nuget package, it already has a PredicateBuilder

Reply:
Download